feature-libs/product-configurator/rulebased/components/attribute/product-card/configurator-attribute-product-card.component.ts

```diff
@@ -10,40 +10,64 @@ import { Configurator } from '../../../core/model/configurator.model';
 import { FormControl } from '@angular/forms';
 import { Subscription } from 'rxjs';
 
+interface QuantityUpdateEvent {
```

Comment:
should this be exported, for cases where someone needs to use it elsewhere?

Reply:
good idea

Reply:
moved to configurator-form.events.ts